snitun/multiplexer/const.py

```diff
@@ -1,11 +1,15 @@
 """This file contains the constants used by the multiplexer."""
 
-INCOMING_QUEUE_MAX_BYTES_CHANNEL = 1024 * 1024 * 10
-INCOMING_QUEUE_LOW_WATERMARK = 1024 * 512 * 1
-INCOMING_QUEUE_HIGH_WATERMARK = 1024 * 1024 * 4
+# When downloading a file, the message size will be
+# ~4199990 bytes which is the protocol maximum. Make
+# sure we have enough space to store 16 messages
```

Comment:
Note that previous maximum was 8000 messages so this is still a bit conservative https://github.com/NabuCasa/snitun/blob/b72e8b65df4509e4627f34b4cf59477a9c2eac80/snitun/multiplexer/channel.py#L37